### PR TITLE
Adding content.feature scenarios using gherkin

### PR DESCRIPTION
**Pull request by pritalpatel**
Sending content.feature scenario in separate PR from #18

### CODE REVIEW

File: tests/_support/Step/Acceptance/Administrator/Content.php [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+		$I = $this;
+		// iterate over all rows
+		foreach ($fields->getRows() as $index => $row) {
+			if ($index === 0) { // first row to define fields
+				$keys = $row;
+				continue;
+			}
+			else
+			{
+				if ($row[0] == "title")
```
Comment (yvesh) [inline tests/_support/Step/Acceptance/Administrator/Content.php]: Can you take a look at the comments i made at Puneets PR? There are several issues here, starting from code style to readability ;)

File: RoboFile.php [hunk 2]
```diff
@@ -278,6 +278,15 @@ public function runTests($opts = ['use-htaccess' => false, 'env' => 'desktop'])
...
 			->arg('--env ' . $opts['env'])
+			->arg('tests/acceptance/content.feature')
+			->run()
+			->stopOnFail();
+
+		$this->taskCodecept($pathToCodeception)
+			->arg('--steps')
+			->arg('--debug')
+			->arg('--fail-fast')
+			->arg('--env ' . $opts['env'])
```
Comment (javigomez) [inline RoboFile.php:289]: interesting ^_^

**Comment by javigomez** [comment]
closing for #27

File: tests/_support/AcceptanceTester.php [hunk 3]
```diff
@@ -12,15 +15,226 @@
...
+				$I->click(['xpath' => "//div[@id='toolbar-unblock']//button"]);
+				break;
+			case "delete":
+				$I->click(['xpath' => "//div[@id='toolbar-delete']//button"]);
+				break;
+			case "featured":
+				$I->click(['xpath' => "//div[@id='toolbar-featured']//button"]);
+				break;
+		}
+	}
```
Comment (nibra) [inline tests/_support/AcceptanceTester.php:173]: Better: 
$buttons = [
    'new' => ['xpath' => "//div[@id='toolbar-new']//button"],
    'edit' => ['xpath' => "//div[@id='toolbar-edit']//button"],
    ...
];
if (isset($button[$input]))
{
    $I->click($button[$input]);
}

File: tests/_support/AcceptanceTester.php [hunk 4]
```diff
@@ -12,15 +15,226 @@
...
+	 */
+	/*public function verifyAvailableTabs($expectedTabs, $tabsLocator = ['xpath' => "//ul[@id='myTabTabs']/li/a"])
+	{
+		$I = $this;
+		$actualArrayOfTabs = $I->grabMultiple($tabsLocator);
+		$I->comment("Fetch the current list of Tabs in the edit view which is: " . implode(", ", $actualArrayOfTabs));
+		$url = $I->grabFromCurrentUrl();
+		$I->assertEquals($expectedTabs, $actualArrayOfTabs, "Tab Labels do not match on edit view of" . $url);
+		$I->comment('Verify the Tabs');
+	}*/
```
Comment (nibra) [inline tests/_support/AcceptanceTester.php:223]: Don't comment out code. We have source control. If you want to have a reminder, create an issue instead.

**Pull request closed without merging**